x-pack/plugins/endpoint/server/types.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export interface EndpointData {
+  machine_id: string;
+  created_at: Date;
+  host: {
+    name: string;
+    hostname: string;
+    ip: string;
+    macAddress: string;
+    os: {
+      name: string;
+      full: string;
+    };
+  };
+  endpoint: {
+    domain: string;
+    is_base_image: boolean;
+    active_directory_distinguishedName: string;
```

Comment:
Make the camelization v. snake casing consistent, same goes for the rest of the fields.

Reply:
Typos from modifying the document.